Library/Homebrew/api.rb

```diff
@@ -0,0 +1,39 @@
+# typed: false
+# frozen_string_literal: true
+
+require "api/analytics"
+require "api/bottle"
+require "api/cask"
+require "api/formula"
+require "api/versions"
+
+module Homebrew
+  # Helper functions for using Homebrew's formulae.brew.sh API.
+  #
+  # @api private
+  module API
+    extend T::Sig
+
+    module_function
+
+    API_DOMAIN = "https://formulae.brew.sh/api"
+
+    sig { params(endpoint: String, json: T::Boolean).returns(T.any(String, Hash)) }
+    def fetch(endpoint, json: false)
```

Comment:
is `json` ever called with a `false` value here? Had a quick look and couldn't find it.

Reply:
No, got a little ahead of myself here. It _will_ be when I add the `/api/cask-source` API because those are ruby files rather than JSON files. I can remove it for now, though, so it's not sitting around unused. Plus, it probably makes more sense to invert the logic and default to converting to JSON unless otherwise specified.